src/skins/blau.tsx

```diff
@@ -167,6 +170,22 @@ export const getBlauSkin: GetSkin = () => {
             textNavigationSearchBarText: palette.white,
             textAppBar: palette.grey5,
             textAppBarSelected: palette.blauBlueSecondary60,
+
+            // TAGS
+            tagBackgroundSuccess: palette.blauGreen10,
+            tagBackgroundWarning: palette.blauYellow10,
+            tagBackgroundError: palette.blauRed10,
+            tagBackgroundPromo: palette.blauPurple10,
+            tagBackgroundActive: palette.blauBlueSecondary10,
+            tagBackgroundInactive: palette.grey2,
+            tagBackgroundInverse: palette.white,
```

Comment:
@yceballost this color (`tagBackgroundInverse`) is defined in the tags spec doesn't exist in the schema
https://github.com/Telefonica/mistica-design/blob/production/tokens/vivo-skin-schema.json

Reply:
my mistake, use `inverse`. Specs corrected!

Reply:
done